components/app-gateway/12-interpolated-defaults.tf

```diff
@@ -76,8 +77,19 @@ data "azurerm_key_vault_secret" "root_certificates" {
   key_vault_id = "/subscriptions/${var.subscription_id}/resourceGroups/${local.resource_group_name}/providers/Microsoft.KeyVault/vaults/crime-portal-kv-${var.env}"
 }
 
-data "azurerm_key_vault_secret" "ssl_certificates" {
-  for_each     = { for ssl_cert in local.ssl_certificates : ssl_cert.ssl_cert_key => ssl_cert }
-  name         = each.value.ssl_cert_name
-  key_vault_id = "/subscriptions/${var.subscription_id}/resourceGroups/${local.resource_group_name}/providers/Microsoft.KeyVault/vaults/crime-portal-kv-${var.env}"
+# data "azurerm_key_vault_secret" "ssl_certificates" {
+#   for_each     = { for ssl_cert in local.ssl_certificates : ssl_cert.ssl_cert_key => ssl_cert }
+#   name         = each.value.ssl_cert_name
+#   key_vault_id = "/subscriptions/${var.subscription_id}/resourceGroups/${local.resource_group_name}/providers/Microsoft.KeyVault/vaults/crime-portal-kv-${var.env}"
+# }
```

Comment:
Could you remove this please? We can get it back from git history if we need it.